velox/experimental/cudf/exec/CudfHashJoin.cpp

```diff
@@ -160,19 +185,35 @@ void CudfHashJoinBuild::noMoreInput() {
     }
   };
 
+  if (CudfConfig::getInstance().debugEnabled) {
+    LOG(INFO) << "CudfHashJoinBuild: build batches" << std::endl;
+    LOG(INFO) << "Build batches number of columns: "
+              << inputs_[0]->getTableView().num_columns() << std::endl;
+    for (auto i = 0; i < inputs_.size(); i++) {
+      LOG(INFO) << "Build batch " << i
+                << ": number of rows: " << inputs_[i]->getTableView().num_rows()
```

Comment:
May override more function in CudfVector, such as childrenSize, or throw exception to make CudfVector functions valid in the future.